README.md

```diff
@@ -59,11 +59,33 @@ $ make cluster/clean
 
 ## VPC Peering 
 Currently AWS resources are deployed into a separate Virtual Private Cloud (VPC) than the VPC that the cluster is deployed into. In order for these to communicate, a `peering connection` must be established between the two VPCS. To do this:
-- In the AWS VPC console, create a new peering connection between the two VPCs. This is a two-way communication channel, so only one needs to be created
-- Select the newly created connection, then click `actions > accept request` to accept the peering request
-- Edit the cluster VPC route table. Create a new route that contains the resource/default VPC's CIDR block as the `destination` and the newly created peering connection as the `target`
-- Edit the resource VPC's route table. Create a new route that contains the CIDR block of the cluster VPC as the `destination` and the peering connection as the `target`. 
-- Edit the Security Groups associated with each VPC to ensure database and cache traffic can pass between the two VPCs.
+1. Create a new peering connection between the two VPCs.
+  - Go to `VPC` > `Peering Connections`
+  - Click on `Create Peering Connection`
+    - **NOTE**: This is a two-way communication channel so only one needs to be created.
+  - Select the newly created connection, then click `Actions` > `Accept Request` to accept the peering request.
+2. Edit the cluster VPC route table
+  - Go to `VPC` > `Your VPCs`
+  - Find the cluster VPC and click on its route table under the heading `Main Route Table`
+    - The cluster VPC usually has an empty name or is called `default`
```

Comment:
The cluster VPC is actually the one with your name, and the resources VPC is usually the default one because resources get provisioned there by default

Reply:
if this is actually your VPC, is the destination/target correct? I remember setting the destination/target to point to these for the `default` VPC.

Reply:
Ah ignore the comment above, i got confused which CIDR block was which :) Think this is right :+1: